src/libraries/Common/tests/System/Net/RemoteExecutorExtensions.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.Tasks;
+using Microsoft.DotNet.RemoteExecutor;
+
+namespace Microsoft.DotNet.RemoteExecutor;
+
+internal static class RemoteExecutorExtensions
+{
+    public static Task DisposeAsync(this RemoteInvokeHandle handle) =>
```

Comment:
Note: [`IAsyncDisposable.DisposeAsync`](https://learn.microsoft.com/en-us/dotnet/api/system.iasyncdisposable.disposeasync) returns a `ValueTask`. If we expect https://github.com/dotnet/arcade/issues/14807 to be addressed by implementing the interface, the signature will change.